admin/company/organizations.md

```diff
@@ -20,6 +20,17 @@ When you have a [self-serve](../../subscription/details.md#self-serve) subscript
 3. Under **Organizations**, select **Overview**.
 4. Select the action icon in the organization's card, and then select **Get more seats**.
 
+
+## Add organizations to a company
+
+There is no limit to the number of organizations you can have under a company layer. All organizations must have Business subscription.
```

Comment:
I noticed this is phrased the same way in the [Admin and Security docs](https://deploy-preview-17640--docsdocker.netlify.app/docker-hub/new-company/), but wonder if this sounds less casual (if you agree, we can update that other page too): 
```suggestion
There is no limit to the number of organizations you can have under a company layer. All organizations must have a Business subscription.
```